### ISSUE TITLE
Insufficient text-to-background contrast ratio on multiple pages

### ISSUE THREAD

**Issue by hallabalint** [zmnnm/lisy-dark#3]
There is an insufficient contrast ratio between the text and the background on several views, which negatively impacts readability and accessibility.

Affected pages/components:
- Beállítások
- Szállítólevél részletei
- Eszköz részletei

**Comment by hallabalint** [comment]
It can also be a problem in Hibajegyek Javítás állapota column
